style.css

```diff
@@ -0,0 +1,347 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap');
+
+html {
+  background: white;
+}
+
+body {
+  background-size: 100%;
+  background-repeat: no-repeat;
+  width: fit-content;
+  background-color: #e5e5e5;
+}
```

Comment:
Please, follow the next steps, those will help you to center your content

- [ ]  Please, add `margin: 0`  in your body.
- [ ] In your ".work-card" use `margin: 20px 0` instead of `20px 10px`.
- [ ] In your ".works" use `padding: 1rem` instead of `margin: 10px`

Reply:
- This has been resolved.